cmdx/config.go

```diff
@@ -51,17 +52,26 @@ func (c *Config) Defaults(cfg interface{}) {
 func (c *Config) Init(cfg interface{}) error {
 	defaults.SetDefaults(cfg)
 
+	if fileExist(c.filename) {
+		return errors.New("config file already exists")
+	}
+
+	return c.Write(cfg)
+}
+
+func (c *Config) Read() (string, error) {
+	cfg, err := os.ReadFile(c.filename)
+	return string(cfg), err
+}
+
+func (c *Config) Write(cfg interface{}) error {
```

Comment:
@spy16 do we want to override whole cfg or would it be better to take just key and this method can update it?

Reply:
It will be more changes. And I think it will be inconsistent with how `Init()` & `Load()` are as well. They both take full config struct.

Reply:
If write a single config, maybe better to be `WriteKey(key string, value interface{}) error` ?

Reply:
or "Set" would be better.

Reply:
@spy16 let's add another method `Set(key string, value interface{}) error` ?

Reply:
I think it's better to keep it consistent (symmetric) with other methods for now. `Load(cfg), Init(cfg) and Write(cfg)`